pkg/storage/stores/shipper/indexshipper/tsdb/sharding/sharding.go

```diff
@@ -0,0 +1,101 @@
+package sharding
```

Comment:
nit: I'd rename the files in this pkg to be more informative. E.g. I'd rename `sharding.go` to `fingerprintsize.go`  or similar.